frontend/app/templates/components/daterange-edit.hbs

```diff
@@ -24,7 +24,7 @@
       <th class="align-middle" scope="row">
         {{t "daterange-edit.to"}}
       </th>
-      <td>
+      <td id="test">
```

Comment:
I am assuming you are using this for a test but nevertheless this seems a bit out of place, maybe give a more telling name?